Packs/Kenna/Integrations/KennaV2/KennaV2.py

```diff
@@ -198,6 +199,22 @@ def get_connectors(client: Client, *_) -> Tuple[str, Dict[str, Any], List[Dict[s
     return human_readable_markdown, context, connectors
 
 
+def inactivate_asset(client: Client, args: dict) -> Tuple[str, Dict[str, Any], List[Dict[str, Any]]]:
+    asset_id = args.get('asset_id')
```

Comment:
If the argument is `required: true` in the yml file, then we don't need to use `get` because the argument will surely be in the dictionary,
So instead of `get` we will use square brackets.
```suggestion
    asset_id = args['asset_id']
```